alerting/src/utils/raster-utils.ts

```diff
@@ -1,4 +1,4 @@
-import { formatUrl } from './server-utils';
+import formatUrl from '../../../packages/format-url';
```

Comment:
Once "packages" becomes "common" and is built as an independent package, we should ideally be able to do the import like this instead: `import formatUrl from 'common/format-url'`;

We have a similar structure for our platform `ovio.org`, I can give you access to the code if it helps @DanielJDufour

Reply:
I reworked things per your suggestion with a slight modification.  I updated the package.json to install the `packages/common` as `prism-common`.  I'm thinking it will help clarify for other developers where the package is coming from and that it is not https://www.npmjs.com/package/common.  Happy to change to just `"common"` if you prefer that though.